crates/bitwarden-wasm/src/client.rs

```diff
@@ -37,11 +37,9 @@ impl BitwardenClient {
     #[wasm_bindgen(constructor)]
     pub fn new(settings_input: Option<String>, log_level: Option<LogLevel>) -> Self {
         console_error_panic_hook::set_once();
-        if let Err(e) =
-            console_log::init_with_level(convert_level(log_level.unwrap_or(LogLevel::Info)))
-        {
-            panic!("failed to initialize logger: {:?}", e);
-        }
+        // This will only fail if another logger was already initialized, so we can ignore the
+        // result.
+        let _ = console_log::init_with_level(convert_level(log_level.unwrap_or(LogLevel::Info)));
```

Comment:
I wonder if maybe it would be better to log warn when this happens, instead of ignoring it, since it potentially reduces a chance for a customer raising an issue about log level not being set.

Reply:
Great question, I will look into this more.

Reply:
Findings: 
  The first log level set with the first client is honored, all subsequent client creation logging levels are ignored.
  
  I added a warning as suggested here. https://github.com/bitwarden/sdk/pull/935/commits/2da01df2393596b7205c708ad36905ab6f0d4f33